### ISSUE TITLE
CalendarView in HC has incorrect cell background color

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#1436]
**Describe the bug**

In HC, calendar date cells are using the wrong background color. For example, in High Contrast Black theme the backgrounds should be black but are currently white.

**Steps to reproduce the bug**

1. Turn on HC
2. Open the CalendarView page in Xaml Controls Gallery

**Expected behavior**

Cell backgrounds should be black in HC 1, HC 2, and HC Black; white in HC White.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/66676068-8cae0e00-ec1b-11e9-9a79-af399d3faf65.png)
(Note that the out of scope cells look correct, the in scope cells look wrong)

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/23682899).
